kotlinpoet/src/main/java/com/squareup/kotlinpoet/FileSpec.kt

```diff
@@ -398,6 +413,28 @@ public class FileSpec private constructor(
       memberImports += Import(memberName.canonicalName, `as`)
     }
 
+    /** Adds a default import for the given [packageName]. */
```

Comment:
Should we include examples of supported inputs, i.e. is `kotlin.*` supported, or should it be `kotlin`? Also, should we support imports for individual types?

Reply:
I thought about individual types but opted against it, I couldn't find any such examples in Kotlin, Java, or Gradle. We can add it if folks come up with one?

Reply:
7159948

Reply:
Nice, thanks, the examples are great. Agree let's keep the scope small until there's a need to support single types.